packages/react-core/src/demos/Wizard/WizardDemo.md

```diff
@@ -229,3 +393,206 @@ class FullPageWizard extends React.Component {
+      </DrawerPanelContent>
+    );
+
+    const configSubstepAPanelContent = (
+      <DrawerPanelContent colorVariant={DrawerColorVariant.light200}>
+        <DrawerHead>
+          <span tabIndex={0} ref={this.drawerRef}>
+            Configuration substep A content
+          </span>
+        </DrawerHead>
+      </DrawerPanelContent>
+    );
+
+    const configSubstepBPanelContent = (
+      <DrawerPanelContent colorVariant={DrawerColorVariant.light200}>
+        <DrawerHead>
+          <span tabIndex={0} ref={this.drawerRef}>
+            Configuration substep B content
+          </span>
+        </DrawerHead>
+      </DrawerPanelContent>
+    );
+
+    const additionalPanelContent = (
+      <DrawerPanelContent colorVariant={DrawerColorVariant.light200}>
+        <DrawerHead>
+          <span tabIndex={0} ref={this.drawerRef}>
+            Additional panel content
+          </span>
+        </DrawerHead>
+      </DrawerPanelContent>
+    );
+
+    const reviewPanelContent = (
+      <DrawerPanelContent colorVariant={DrawerColorVariant.light200}>
+        <DrawerHead>
+          <span tabIndex={0} ref={this.drawerRef}>
+            Review panel content
+          </span>
+        </DrawerHead>
+      </DrawerPanelContent>
+    );
+
+    const steps = [
+      { id: 0, name: 'Information', component: <p>Step 1 content</p>, drawerPanelContent: informationPanelContent },
+      {
+        id: 1,
+        name: 'Configuration',
+        steps: [
+          {
+            id: 2,
+            name: 'Substep A',
+            component: <p>Configuration substep A</p>,
+            drawerPanelContent: configSubstepAPanelContent
+          },
+          {
+            id: 3,
+            name: 'Substep B',
+            component: <p>Configuration substep B</p>,
+            drawerPanelContent: configSubstepBPanelContent
+          }
+        ]
+      },
+      { id: 4, name: 'Additional', component: <p>Step 3 content</p>, drawerPanelContent: additionalPanelContent },
+      {
+        id: 5,
+        name: 'Review',
+        component: <p>Review step content</p>,
+        nextButtonText: 'Finish',
+        drawerPanelContent: reviewPanelContent
+      }
+    ];
+    const title = 'Basic wizard';
+
+    return (
+      <React.Fragment>
+        <Page
+          header={Header}
+          sidebar={Sidebar}
+          isManagedSidebar
+          skipToContent={PageSkipToContent}
+          breadcrumb={PageBreadcrumb}
+          mainContainerId={pageId}
+        >
+          <PageSection variant={PageSectionVariants.light}>
+            <TextContent>
+              <Text component="h1">Main title</Text>
+              <Text component="p">A demo of a wizard in a page.</Text>
+            </TextContent>
+          </PageSection>
+          <PageSection type={PageSectionTypes.wizard} variant={PageSectionVariants.light}>
+            <Wizard hasDrawer navAriaLabel={`${title} steps`} mainAriaLabel={`${title} content`} steps={steps} />
```

Comment:
Applying an `aria-label` here `mainAriaLabel={`${title} content`}` results in an Axe error. 

`aria-label attribute is not well supported on a div with no valid role attribute`

I think you can safely remove it.